gridstatus/nyiso.py

```diff
@@ -472,7 +509,7 @@ def get_interconnection_queue(self, verbose=False):
         }
 
         extra_columns = [
-            "Proposed  In-Service",
+            "Proposed In-Service",
```

Comment:
same comment as above

should this be "Proposed In-Service Date" since that is the value in active?